Fabric.Authorization.Persistence.SqlServer/EntityModels/GroupRole.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Fabric.Authorization.Domain.Models;
+
+namespace Fabric.Authorization.Persistence.SqlServer.EntityModels
+{
+    public class GroupRole : ITrackable, ISoftDelete
```

Comment:
Is this an explicit join entity? Does EF core not support many to many relationships with modelling the explicit join entity? Maybe a better question is, what are the advantages vs disadvantages of an explicit join entity?

Reply:
Just to close the loop in this thread, below are 2 articles referencing EF Core many to many relationships. I do need to adjust my models to reference the join/linking entity as well. I will do that.

https://docs.microsoft.com/en-us/ef/core/modeling/relationships
Many-to-many relationships without an entity class to represent the join table are not yet supported. However, you can represent a many-to-many relationship by including an entity class for the join table and mapping two separate one-to-many relationships.

http://www.thereformedprogrammer.net/updating-many-to-many-relationships-in-entity-framework-core/